main.go

```diff
@@ -109,26 +110,41 @@ func main() {
 	submSpec.CableDriver = strings.ToLower(submSpec.CableDriver)
 
 	localEndpoint, err := util.GetLocalEndpoint(submSpec.ClusterID, submSpec.CableDriver, map[string]string{}, submSpec.NatEnabled,
-		localSubnets, util.GetLocalIP())
+		localSubnets, util.GetLocalIP(), submSpec.ClusterCidr)
 
 	if err != nil {
 		klog.Fatalf("Error creating local endpoint object from %#v: %v", submSpec, err)
 	}
 
+	klog.Errorf("The local endpoint is %v", localEndpoint)
 	cableEngine := cableengine.NewEngine(localCluster, localEndpoint)
 
+	err = subv1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		klog.Errorf("Error adding submariner types to the scheme: %v", err)
+	}
+
+	var cableHealthchecker *healthchecker.HealthChecker
+	if len(submSpec.GlobalCidr) == 0 {
```

Comment:
This will break the "one slide globalnet" , another side not globalnet usecase we found for the storage cluster. 

But we can fix it by adding globalnet support as a bugfix ;)

Reply:
yes, we can fix it, once the datapath changes are ready.